pkg/apis/config/defaults/default.go

```diff
@@ -54,4 +54,7 @@ const (
 
 	MdnsDiscovery DiscoveryType = "MDNS"
 	DhtDiscovery  DiscoveryType = "DHT"
+
+	// Allowed values for DefaultServiceFilterMode are "FilterIfLabelDoesNotExists" and "FilterIfLabelExists"
+	DefaultServiceFilterMode string = "FilterIfLabelExists"
```

Comment:
Users don't know what DefaultServiceFilterMode means. A better implementation is as follows:
```
type ServiceFilterMode string

FilterIfLabelExistsMode        ServiceFilterMode = "FilterIfLabelExists"
FilterIfLabelDoesNotExistsMode ServiceFilterMode = "FilterIfLabelDoesNotExists"
```

Reply:
done